backend/clap/proto/data_store.proto

```diff
@@ -0,0 +1,106 @@
+// Proto file from the shared folder
+syntax = "proto3";
+
+package data_store;
+
+option java_multiple_files = true;
+option java_outer_classname = "DataStoreProto";
+option go_package = "github.com/Veermove/zpi/data-store";
+
+enum ProjectState {
+    DRAFT = 0;
+    SUBMITTED = 1;
+    ACCEPTED = 2;
+    REJECTED = 3;;
+}
+
+enum RatingType {
+    INDIVIDUAL = 0;
+    INITIAL = 1;
+    FINAL = 2;
+}
+
+message SubmissionRequest {
+    int32 contest_id = 1;
+}
+
+message DetailsSubmissionRequest {
+    int32 submission_id = 1;
+}
+
+message RatingsSubmissionRequest {
+    int32 submission_id = 1;
+    int32 constest_id   = 2;
```

Comment:
Do we need `contest_id`? `submission_id` is enough to identify a specific submission.

And, if we need, there's a spelling mistake.

Reply:
i assumed we want to show ratings only for current/active contest.